utils/http.go

```diff
@@ -97,7 +97,7 @@ func (r *RequestResponse) Resolve(key string) interface{} {
 }
 
 func (r *RequestResponse) Default() interface{} {
-	return r.JSON()
+	return r
```

Comment:
I think we've talked about having JSON serialization be explicit, via something like a `JSON()` method